examples/provider/bid/audience-network/README.md

```diff
@@ -0,0 +1,46 @@
+# audience-network-bid-delegate
+
+BidProvider for Audience Network
+
+## Usage
+
+```html
```

Comment:
@elljoh - I've whitelisted your test app for AN Header Bidding - can you retest please?